Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
the error seems to be a legit error after #474 , grey_allOf accepting varargs but grey_allOfMatchers accepting an array, is it not the case here?